### PR TITLE
Add remove vowels from a string algorithm

### PR DESCRIPTION
**Pull request by jsca-kwok**
This PR proposes to add 3 solutions to the remove vowels challenge:
- Brute force
- Regex
- Ruby .delete() method

### CODE REVIEW

File: data_structures/arrays/remove_vowels.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+# @return {String}
+
+#
+# Approach 1: Brute Force
+#
+# Time Complexity: O(n)
+#
+def remove_vowels(s)
+  result_array = []
+  s.downcase
```
Comment (vzvu3k6k) [inline data_structures/arrays/remove_vowels.rb:23]:
```suggestion
  s.downcase!
```

Did you mean `downcase!`?

File: data_structures/arrays/remove_vowels.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    if letter != 'a' && letter != 'e' && letter != 'i' && letter != 'o' && letter != 'u'
+      result_array.push(letter)
+    end
+  end
+
+  result_array.join('')
+end
+
+s = 'leetcodeisacommunityforcoders'
+print(remove_vowels(s))
```
Comment (vzvu3k6k) [inline data_structures/arrays/remove_vowels.rb:36]:
```suggestion
puts(remove_vowels(s))
```

`puts` would be better because it prepends a newline.

**Review by vzvu3k6k — COMMENTED** [review]
Some solutions seems to have nothing to do with arrays, so there might be a more appropriate directory than `array`.

But anyway it looks mostly good to me :+1:

**Comment by jsca-kwok** [comment]
> Some solutions seems to have nothing to do with arrays, so there might be a more appropriate directory than `array`.
> 
> But anyway it looks mostly good to me 👍

Oh yes, good point! Do you have a suggestion for where I should move those solutions to? Perhaps just "other"?

**Comment by vbrazo** [comment]
Some of your solutions use Ruby methods, and whenever you're using them, you need to be able to explain how they work internally (i.e: their time complexity). If you can't, ideally you should just do your own loops and use the array techniques: two-pointers and in-place algorithms when needed. In some cases, a technique is not required because the question is a bit simpler.

**Comment by vbrazo** [comment]
@jsca-kwok I think this is a string question. I know it can end up in an array but I'd categorize this as string.

File: strings/remove_vowels.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Challenge name: Remove vowels from a string
```
Comment (vbrazo) [inline strings/remove_vowels.rb:1]: Ruby#String: https://apidock.com/ruby/String/delete.

**Review by vbrazo — APPROVED** [review]

**Comment by vbrazo** [comment]
Array is one of the fundamental blocks in data structure. Since a string is just formed by an array of characters, they are both similar. Some questions fall into this category and I thought it was appropriate to create `arrays/strings` folder.

**Pull request merged**